docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+* `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command
+* `destination-dir` is where Machine ID writes renewable certificates, which are only used by Machine ID and should not be used by applications and tools
+* `data-dir` is where Machine ID writes the short-lived certificate. This certificate should be used by applications and tools.
+<Details scope={["cloud"]} scopeOnly={true}>
+* `auth-server` is the address of your Teleport Auth Server
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+* `auth-server` is the address of your Teleport Cloud Proxy Server
+</Details>
+
+Now that Machine ID has successfully started, let's investigate the
+`/opt/machine-id` directory to see what was written to disk.
+
+```
+$ tree /opt/machine-id
+machine-id
+├── key
+├── key.pub
+├── known_hosts
+├── ssh_config
+├── sshcacerts
+├── sshcert
+├── tlscacerts
+└── tlscert
+
+0 directories, 8 files
+```
+
+This directory contains private key material in the `key.*` files, SSH
+certificates in the `ssh*` files, X.509 certificates in the `tls*` files, and
+OpenSSH configuration in the `ssh_config` and `known_hosts` files to make it easy
+to integrate Machine ID with external applications and tools.
+
+## Step 4/4. Use certificates issued by Machine ID
+
+To use Machine ID, find a host that you want to connect to within your cluster
+using `tsh ls`. You might see output like the following on your system.
+
+```code
+$ tsh ls
+Node Name Address        Labels                        
+--------- -------------- ----------------------------- 
+node-name  127.0.0.1:3022 arch=x86_64,group=api-servers 
+```
+
+To use Machine ID with the OpenSSH integration, run the following command to
+connect to `node-name` within cluster `example.com`.
+
+```
+ssh -F /opt/machine-id/ssh_config root@node-name.example.com
+```
+
+<Admonition type="note" title="Roles must have logins defined">
+  If you see the below error, it means the user you are trying to log in as is
+  not specified under `logins` in the role you are using. If you have been
+  following along with the `access` role, run
+  `tctl get roles/access > access.yaml`, update the `logins` field, and update
+  your role with `tctl create -f access.yaml`.
+
+  ```code
+  $ ssh -F /opt/machine-id/ssh_config root@node-name
+  root@node-name: Permission denied (publickey).
+  kex_exchange_identification: Connection closed by remote host
+  ```
+</Admonition>
+
+Now you can replace any invocations of `ssh` with the above command to provide
+your applications and tools a machine identity that can be rotated, audited,
+and controlled with all the familiar Teleport access controls.
+
+{
+/*
+TODO(russjones): Once we have other links, include them here.
+
+## Next Steps
+
+For the next steps, dive deeper into the topics relevant to your Machine ID use-case, for example:
+
+* Check out configuration [guides](./guides.mdx).
+* See [frequently asked questions](./faq.mdx).
```

Comment:
Once they're done, I think we should link the Jenkins and Ansible guides in this section. @russjones what do you think?

Reply:
Sure, will do.